Flask_app/models/models.py

```diff
@@ -64,4 +67,17 @@ def verify_password(self, user_password):
 
 
     def __repr__(self) -> str:
-        return '<User %r>' % self.email
\ No newline at end of file
+        return '<User %r>' % self.email
```

Comment:
Explain me the use of this method

Reply:
It shows what the email address of the user object is. It helps in debugging and writing code.